clearpath_hardware_interfaces/scripts/light_emulator.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import tkinter as tk
```

Comment:
Should the be a dep?  If so, will it install any graphical deps?

Reply:
That script is just intended to test the lighting node without a robot. It was used to generate the GIFs here https://docs.clearpathrobotics.com/docs/ros/api/lighting/. It does require a GUI which is why I didn't add any dependencies for it.

This might not be the right spot for it, maybe we should make a separate repo with tools like this.

Reply:
Splitting it out into a separate package with complete dependencies might make the most sense? It can probably stay in this repo, but we probably shouldn't have packages with intentionally-omitted dependencies IMO

Reply:
Removed the script for now.